datalad/support/gitrepo.py

```diff
@@ -157,6 +157,26 @@ def newfunc(self, files, *args, **kwargs):
     return newfunc
 
 
+def _fix_paths_list(files):
```

Comment:
in general  this  one  has nothing to do with paths.  it is re remove_empty_items  or alike